Bio/pairwise2.py

```diff
@@ -1160,7 +1160,12 @@ def format_alignment(align1, align2, score, begin, end):
 # Try and load C implementations of functions. If I can't,
 # then throw a warning and use the pure Python implementations.
 # The redefinition is deliberate, thus the no quality assurance
-# flag for when using flake8:
+# flag for when using flake8.
+# Before, we secure access from outside to the Python functions:
+
+python_make_score_matrix_fast = _make_score_matrix_fast
+python_rint = rint
```

Comment:
Give these leading underscores as implied private, or do you deliberately want them in the public API?

Reply:
Good point, will change